### ISSUE TITLE
Problem with controlnet

### ISSUE THREAD

**Issue by OoAnikioO** [lllyasviel/stable-diffusion-webui-forge#1837]
I made this post a month ago:
https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/1140

the problem is still present, so I cannot work with forge

I add the log: is it due to the message :   "DeprecationWarning: pkg_resources is deprecated as an API"  ??
if yes, what should I do? I already tried to re-install

Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
Version: f2.0.1v1.10.1-previous-531-g210af4f8
Commit hash: 210af4f80406f78a67e1c35a64a6febdf1200a82
E:\StableDiffusion\Packages\Stable Diffusion WebUI Forge\extensions\sd-webui-infinite-image-browsing\install.py:3: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  import pkg_resources
E:\StableDiffusion\Packages\Stable Diffusion WebUI Forge\extensions-builtin\forge_legacy_preprocessors\install.py:2: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  import pkg_resources
E:\StableDiffusion\Packages\Stable Diffusion WebUI Forge\extensions-builtin\sd_forge_controlnet\install.py:2: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  import pkg_resources
Launching Web UI with arguments: --cuda-malloc --gradio-allowed-path 'E:\StableDiffusion\Images'
Using cudaMallocAsync backend.
Total VRAM 8192 MB, total RAM 32680 MB
pytorch version: 2.3.1+cu121
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 2060 SUPER : cudaMallocAsync
VAE dtype preferences: [torch.float32] -> torch.float32
CUDA Using Stream: False
Using pytorch cross attention
Using pytorch attention for VAE
ControlNet preprocessor location: E:\StableDiffusion\Packages\Stable Diffusion WebUI Forge\models\ControlNetPreprocessor
2024-09-16 13:27:08,909 - ControlNet - INFO - ControlNet UI callback registered.
Model selected: {'checkpoint_info': {'filename': 'E:\\StableDiffusion\\Packages\\Stable Diffusion WebUI Forge\\models\\Stable-diffusion\\Flux\\flux1-dev-bnb-nf4-v2.safetensors', 'hash': 'f0770152'}, 'additional_modules': [], 'unet_storage_dtype': None}
Using online LoRAs in FP16: False
Running on local URL:  http://127.0.0.1:7861

To create a public link, set `share=True` in `launch()`.
IIB Database file has been successfully backed up to the backup folder.
Startup time: 84.0s (prepare environment: 23.7s, launcher: 0.2s, import torch: 43.7s, initialize shared: 0.9s, other imports: 4.4s, list SD models: 1.3s, load scripts: 6.0s, initialize extra networks: 0.1s, create ui: 2.9s, gradio launch: 1.4s, app_started_callback: 0.1s).

**Comment by lunar-studio** [comment]
I have the same deprecated warnings but my forge works regardless. I don't know if this has any impact.

**Comment by DenOfEquity** [comment]
I don't get those deprecation warnings, but they shouldn't be relevant.
I do get, as expected, identical results in img2img and inpaint upload. The depth map control image is correct for the source, and generations follow the control image. Are you using low Control Weight or Timestep Range? Or a mask?

**Comment by OoAnikioO** [comment]
> I don't get those deprecation warnings, but they shouldn't be relevant. I do get, as expected, identical results in img2img and inpaint upload. The depth map control image is correct for the source, and generations follow the control image. Are you using low Control Weight or Timestep Range? Or a mask?

i use a mask
